src/common/saveEdits.js

```diff
@@ -0,0 +1,10 @@
+const saveEdits = (e) => {
+  e.stopPropagation();
+  const text = e.currentTarget.innerHTML;
+  const index = e.currentTarget.parentNode.childNodes[0].innerHTML;
+  const toDolist = JSON.parse(localStorage.getItem('toDolist'));
+  toDolist[index].description = text;
+  localStorage.setItem('toDolist', JSON.stringify(toDolist));
+};
+/* eslint-disable */
+export { saveEdits };
```

Comment:
- [ ] As I mentioned in the previous comment(s), Kindly make sure you fix all your `linters` errors instead of disabling them as you did for best practices and clean code.

Reply:
Changed the same.